runtime/src/main/java/com/dylibso/chicory/runtime/InterpreterMachine.java

```diff
@@ -27,10 +33,26 @@ public class InterpreterMachine implements Machine {
 
     private final Instance instance;
 
+    private final Stratum stratum;
+
     public InterpreterMachine(Instance instance) {
         this.instance = instance;
         stack = new MStack();
         this.callStack = new ArrayDeque<>();
+
+        CustomSection smapSection = instance.module().customSection("SMAP");
```

Comment:
- maybe let's add a few notes about the convention we are exploiting here, I see that this custom section SMAP is being inserted by us in Generator.java;
- this also means that the module is a "meta-module", that is it won't contain any code; 
- thus, it doesn't even make a lot of sense for it to be interpreted in the first place